src/main/asciidoc/appendix/graphs.adoc

```diff
@@ -34,7 +34,15 @@ but differ by the underlying objects of vertices and edges:
 
 ArcadeDB is a property graph, which means a vertex or edge consists of an identifier (RID),
 a label (type), and properties (document), in addition to the ordered pairs of endpoints.
-The latter are here vertex properties of incoming and outgoing edges, instead of edge properties of ordered vertex pairs, for technical reasons.
+Practically this means, an edge type has two implicit properties:
+
+* `@in` holding the **RID** of its source vertex,
```

Comment:
It's actually the opposite. The `@out` of the edge is attached to the `@out` of the source vertex. It took me weeks to understand that from Apache TinkerPop Gremlin (Blueprints) when I saw it... it's counterintuitive I know :-(

Reply:
I guessed so before, but I must have been confused by the studio output. I will try to understand this in more depth from the technical and user perspective.